database.go

```diff
@@ -0,0 +1,78 @@
+package authenticatedblockstore
+
+import (
+	"fmt"
+	"log"
+)
+
+type DataBase struct {
+	tableManager *TableManager
+	logManager   *LogManager
+}
+
+// Exported Package APIs.
+// These are called by: package.api()
+//
+//
+func Create() (*DataBase, error) {
+	fmt.Println("Creating a new DataBase, returning a handle to the caller.")
```

Comment:
Probably better to use https://golang.org/pkg/log/.

Reply:
Right, I will turn all of those into logs, thanks.